application/phoenix/README.md

```diff
@@ -0,0 +1,35 @@
+This Phoenix example checks if the environment variables `DEMO_USERNAME` and `DEMO_PASSWORD` have been set. If that's the case, you'll receive a personalized greeting message on http://localhost:4000, otherwise you will get an error message.
```

Comment:
We now have a graphic to use as a heading: https://secrethub.io/img/integrations/phoenix/github-banner.png. See how the other examples do it.

Reply:
Alright, it's added. I will do the same for all the other guides.